server/data/dataAccess/db.js

```diff
@@ -11,7 +11,7 @@ const pool = new Pool({
   password: config.db.password,
   port: config.db.port,
   ssl:
-    config.db.sslEnabled === 'true'
+    config.production && config.db.sslEnabled === 'true'
```

Comment:
we now require db.js in test and mock out db.query in tests, so this actually fires. 
As we don't have a `.env` in CI and `sslEnabled` defaults to `true`, it will try and load a non existent file and fail all persistence tests.